### PR TITLE
Adding setup.sh and pulling in newest project dependencies

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
## Additions
- Created a setup.sh file based off cfpb/cfgov-refresh to pull in the latest project dependencies as folks work on this repo.
- Pulled in changes from cf-layout
## Review
- @anselmbradford

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  echo 'Building project...'
+
+  grunt build
+}
+
+init
+clean
+install
+build
+echo 'Your project is ready. Run jekyll serve to run.'
```
Comment (Scotchester) [inline setup.sh]: Should we also include `bundle install` in this script?

Comment (KimberlyMunoz) [inline setup.sh]: Yep. Included it in the install step in a821900.

Comment (Scotchester) [inline setup.sh]: I'd also suggest that we change the instruction here to read "Run `bundle exec jekyll serve --watch --baseurl ''` to run locally."

Comment (ascott1) [inline setup.sh]: Yes, please. This caught me up yesterday.

Comment (ascott1) [inline setup.sh]: Though it doesn't have to happen as part of this PR

File: README.md [hunk 2]
```diff
@@ -228,8 +228,8 @@ We recommend adding the following checklist to an issue when it's time to publis
...
-- Minicon font - All new minicons must have design approval before they are added to the minicon font. Once approved, an SVG file should be emailed to Daniel Pizarro. Updates to the minicon font are released every month or in an as needed basis. The updated file will be provided in a TrueType format (`.ttf`). 
-- Add to asset library – This is the responsibility of the designer or UX designer that volunteers for the issue. The final asset should either be added to the library or the template available on CFPB’s Google Drive. Examples of assets include the illustration library, isocon library, minicon font, cf.gov web templates, or print suite templates. Changes to the minicon font must go through Daniel Pizarro, are released in batches, and require more lead time. 
+- Minicon font - All new minicons must have design approval before they are added to the minicon font. Once approved, an SVG file should be emailed to Daniel Pizarro. Updates to the minicon font are released every month or in an as needed basis. The updated file will be provided in a TrueType format (`.ttf`).
```
Comment (anselmbradford) [inline README.md:231]: Is any more contact directions than his name needed here for Daniel Pizarro?

File: setup.sh (hunk 1 above)
Comment (anselmbradford) [inline setup.sh]: +1 to above also I'd probably put this `echo` inside and at the end of the `build` function.

File: README.md (hunk 2 above)
Comment (KimberlyMunoz) [inline README.md:231]: I would think that's enough for someone with an approved icon to contact him. We could add an email or Github tag in another PR/issue. What say you @Dnpizarro?

File: setup.sh (hunk 1 above)
Comment (KimberlyMunoz) [inline setup.sh]: Cool. I can do all the above.

**Comment by KimberlyMunoz** [comment]
Ready for another review!

**Comment by anselmbradford** [comment]
:+1:

File: setup.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  npm install
+  bower install --config.interactive=false
+}
+
+# Run tasks to build the project for distribution.
+build(){
+  echo 'Building project...'
+  grunt build
+  echo 'Your project is ready.' 
+  echo 'To serve, run `bundle exec jekyll serve --watch --baseurl ''`'
```
Comment (Scotchester) [inline setup.sh]: Are the nested single quotes going to cause a problem here?

Comment (KimberlyMunoz) [inline setup.sh]: It cuts off early. I can't seem to find a way to escape single quotes, so I'm going to delete this line for now and leave it to a future person to figure out how to include it.

Comment (Scotchester) [inline setup.sh]: Are we too pure to just use double quotes around the whole echoed string? :)

Comment (KimberlyMunoz) [inline setup.sh]: I don't want to risk a back-and-forth on single-versus-double quotes to hold this up.

**Comment by Scotchester** [comment]
:+1:

**Pull request merged**